.eslintrc.json

```diff
@@ -0,0 +1,76 @@
+{
+    "parser": "@typescript-eslint/parser",
+    "parserOptions": {
+        "ecmaVersion": 2018, // Allows for the parsing of modern ECMAScript features
+        "sourceType": "module", // Allows for the use of imports
+        "project": "./tsconfig.json"
+    },
+    "extends": [
+        "eslint:recommended",
+        "plugin:@typescript-eslint/recommended",
+        "plugin:@typescript-eslint/recommended-requiring-type-checking",
+        "plugin:import/errors",
+        "plugin:import/warnings",
+        "plugin:import/typescript"
+    ],
+    "plugins": ["@typescript-eslint", "import"],
+    "reportUnusedDisableDirectives": true,
+    "env": {
+        "node": true
+    },
+    "ignorePatterns": ["**/*.d.ts", "build/**/*"],
```

Comment:
Why would we ignore .d.ts files?
We are using some...

Reply:
adjusted, right ...